test/broadcast.jl

```diff
@@ -352,3 +352,6 @@ StrangeType18623(x,y) = (x,y)
 # Issue 18622
 @test @inferred muladd.([1.0], [2.0], [3.0])::Vector{Float64} == [5.0]
 @test @inferred tuple.(1:3, 4:6, 7:9)::Vector{Tuple{Int,Int,Int}} == [(1,4,7), (2,5,8), (3,6,9)]
+
+# 19419
+@test @inferred broadcast(round, Int, [1])::Vector{Int} == [1]
```

Comment:
With `broadcast(round, Int, [1])` now inferred, drop the `::Vector{Int}`? Also, shouldn't these combined `@test-@inferred` tests be written `@test @inferred(broadcast(round, Int, [1])) == [1]` instead? Otherwise the `@inferred` checks whether the `==` expression is inferable rather than the LHS? Likewise several of the preceding tests. Best!